benefits/eligibility/verify.py

```diff
@@ -3,6 +3,11 @@
 from eligibility_api.client import Client
 
 
+def typenames_to_verify(agency, verifier):
+    """Get the names of the eligibility types to check for the agency/verifier pair."""
+    return [t.name for t in agency.types_to_verify(verifier)]
```

Comment:
Helper gets the names of the types to verify between the `agency` and `verifier` - used in a few places